### PR TITLE
Grid

### PR DESCRIPTION
**Pull request by gamaanderson**
This closes issue #32. There may be some improvements left, but I think they can wait. Notice the need to update for pyart 1.5

### CODE REVIEW

File: artview/components/plot_grid.py [hunk 1]
```diff
@@ -364,14 +377,33 @@ def _add_toolsBoxUI(self):
         toolValueClick = toolmenu.addAction("Click for Value")
         toolSelectRegion = toolmenu.addAction("Select a Region of Interest")
         toolCustom = toolmenu.addAction("Use Custom Tool")
+        toolReset = toolmenu.addAction("Reset Tools")
         toolDefault = toolmenu.addAction("Reset File Defaults")
         toolZoomPan.triggered[()].connect(self.toolZoomPanCmd)
         toolValueClick.triggered[()].connect(self.toolValueClickCmd)
         toolSelectRegion.triggered[()].connect(self.toolSelectRegionCmd)
         toolCustom.triggered[()].connect(self.toolCustomCmd)
```
Comment (nguy) [inline artview/components/plot_grid.py]: Any chance you would remove all reference to 'Custom Tools' in this file and ```plot_radar.py```?
I think it's obsolete with the plugin structure.

File: artview/scripts/grid.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
+"""
+standard.py
```
Comment (nguy) [inline artview/scripts/grid.py]: This should be changed to grid.py

**Comment by nguy** [comment]
@gamaanderson Could you roll the rev sub number with this PR as well?

**Comment by gamaanderson** [comment]
all Done.

**Comment by nguy** [comment]
Thanks!

**Pull request merged**